clients/javascript_client/fma_connect/clients.ts

```diff
@@ -1,3 +1,3 @@
-import { DefaultSettings } from './settings';
+// import { DefaultSettings } from './settings';
```

Comment:
These lines with definitely be needed but commenting these out because of the linter error we get about an unused variable

Reply:
follow-up PR fix I assume?

Reply:
Yeah this is the beginning of the client but there isnt any executable code created yet for this

Reply:
got it